### ISSUE TITLE
bug: table(platform): different width of the heading cells and body cells

### ISSUE THREAD

**Issue by N1XUS** [SAP/fundamental-ngx#7045]
#### Is this a bug, enhancement, or feature request?
bug
#### Briefly describe your proposal.
Platform table has different width for header cells and table body cells:
![image](https://user-images.githubusercontent.com/6586561/139424948-7354329c-1165-426b-9b02-a13eb88fe9ac.png)

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
current
#### If this is a bug, please provide steps for reproducing it.
https://stackblitz.com/edit/angular-itvif2
#### Please provide relevant source code if applicable.
https://stackblitz.com/edit/angular-itvif2

**Comment by platon-rov** [comment]
Actually, StackBlitz has issues not related to the table component, it grabs different ngx version from the registry and so on. It should be resolved in separate pr. But what about dxp here is [the](https://github.com/SAP/fundamental-ngx/pull/7073) fix so you will be unblocked in your work.

**Comment by platon-rov** [comment]
StackBlitz issue https://github.com/SAP/fundamental-ngx/issues/7074.
